### DISCUSSION TITLE
Sunsynk Inverter

### DISCUSSION THREAD

**Discussion by bramuchachos** [StephanJoubert/home_assistant_solarman#331] [General]
Hi,
I have tried to use this integration on my Sunksynk inverter and I am not able to add the data logger because the configuraiton expects the serial number to be all numbers. My data logger serial number is E4701227A085.

Is anyone else having this issue?

**Comment by gurmukhp** [comment]
Have you tried this?

```This must be a string. So if your serial is a number only surround it with quotes '1000'```
